### PR TITLE
Fix:  npm install is broken in assets

### PR DESCRIPTION
**Pull request by aldipower**
Updated npm-shrinkwrap.json in assets to match package.json.
Updated README.md in assets to explain how to add a npm package.

Related to #1109

### CODE REVIEW

**Comment by sttts** [comment]
:+1:

File: src/main/resources/assets/README.md [hunk 1]
```diff
@@ -31,6 +31,22 @@ easy editing, set the `assets_path` flag when running Marathon:
...
 
+#### Adding npm package dependencies to package.json
+
+If you want to add a new npm package to 'node_modules':
+
+1. Edit the dependencies section in package.json and add your new package.
+
+2. Install the new package
+
+	npm install [your package]
```
Comment (mlunoe) [inline src/main/resources/assets/README.md]: `npm install [your package] --save` will install and save to dependencies in `package.json` and `--save-dev` will add it to devDependencies.

Comment (aldipower) [inline src/main/resources/assets/README.md]: :thumbsup:
Thx!

File: src/main/resources/assets/README.md [hunk 2]
```diff
@@ -23,14 +24,34 @@ easy editing, set the `assets_path` flag when running Marathon:
...
+
+        npm install [your package] --save
+    will install and save to dependencies in package.json and
+
+        npm install [your package] --save-dev
+    will add it to devDependencies.
+
+2. Create a synced npm-shrinkwrap.json
+
+        npm shrinkwrap
```
Comment (mlunoe) [inline src/main/resources/assets/README.md]: Missed this the first time. You need to run `npm shrinkwrap --dev` to get the devDependencies shrinkwrapped too.

Comment (mlunoe) [inline src/main/resources/assets/README.md]: Other than that, it looks good :)

Comment (aldipower) [inline src/main/resources/assets/README.md]: Yeah, thanx!

**Pull request merged**